test/FastTests/RavenTestBase.cs

```diff
@@ -170,6 +170,7 @@ private static RavenServer CreateServer(int port)
 
             var doc = MultiDatabase.CreateDatabaseDocument(name);
             doc.Settings["Raven/DataDir"] = path;
+            doc.Settings["Raven/Indexing/ThrowIfAnyIndexCouldNotBeOpened"] = "true";
```

Comment:
Why are you doing this here, if you have a test that uses the modifyConiguration to set it?